packages/common/src/groups/_internal/GroupUiSchemaCreateFollowers.ts

```diff
@@ -115,3 +120,24 @@ export const buildUiSchema = async (
     ],
   };
 };
+
+/**
+ * @private
+ * constructs the default values for creating a followers group.
+ * This is used to pre-populate the form with specific default values
+ * that are different from the normal Group Schema defualts.
+ * @param i18nScope
+ * @param options
+ * @param context
+ * @returns
+ */
+export const buildDefaults = async (
```

Comment:
not that this is your responsibility, but I believe Ben's work merged, so if you wanted to hook up the `buildDefaults` functions in the `GroupUiSchemaCreateEdit` and `GroupUiSchemaCreateView`, that would be great.